pkgs/servers/dns/pdns-recursor/default.nix

```diff
@@ -1,28 +1,25 @@
 { stdenv, fetchurl, pkgconfig, boost
 , openssl, systemd, lua, luajit, protobuf
-, enableLua ? false
 , enableProtoBuf ? false
 }:
-
-assert enableLua      -> lua != null && luajit != null;
 assert enableProtoBuf -> protobuf != null;
 
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "pdns-recursor-${version}";
-  version = "4.0.8";
+  version = "4.1.1";
 
   src = fetchurl {
     url = "https://downloads.powerdns.com/releases/pdns-recursor-${version}.tar.bz2";
-    sha256 = "04v5y6mfdhn8ikigqmm3k5k0zz5l8d3k1a7ih464n1161q7z0vww";
+    sha256 = "0srrw726qpwg69v75dwbxab9hk73x1wia4rcnmf7g5qr2k3h7swg";
   };
 
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [
     boost openssl systemd
-  ] ++ optional enableLua [ lua luajit ]
-    ++ optional enableProtoBuf protobuf;
```

Comment:
Lua is not optional anymore.

Reply:
Uh, to me it's suspicious to have *both* lua and luajit, as they're two different implementations of the same language.

Reply:
Well, I didn't know that. I tried building it with lua only and it succeeds but not with luajit.

lua only:
```
checking whether we will be linking in LuaJIT... no
checking whether we will be linking in Lua... auto
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... yes
checking for chosen LUA... lua
checking lua.hpp usability... yes
checking lua.hpp presence... yes
checking for lua.hpp... yes
[...]
```

luajit only:
```
checking whether we will be linking in LuaJIT... no
checking whether we will be linking in Lua... auto
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for chosen LUA... not found
configure: error: Neither Lua nor LuaJIT found, Lua support is not optional
```
It seems it can't detect it.